bundler.d/assets.rb

```diff
@@ -1,7 +1,7 @@
 gem 'jquery-turbolinks', '~> 2.1'
 gem 'jquery-ui-rails', '< 5.0.0'
 gem 'autoprefixer-rails', '~> 5.2'
-gem 'patternfly-sass', '~> 2.7'
+gem 'patternfly-sass', '~> 3.9'
```

Comment:
can you browse though the patternfly changelog? I wonder if there is any impact.